data/fusemachines.json

```diff
@@ -3,7 +3,7 @@
   "establishedAt": 1356977700,
   "location": "Devkota Sadak, Baneshwor, Kathmandu",
   "website": "https://fusemachines.com/",
-  "email": null,
+  "email": "info@fusemachines.com",
```

Comment:
Updated the email through their contact info : https://fusemachines.com/contact/